overlord/servicestate/servicemgr.go

```diff
@@ -61,9 +61,18 @@ func Manager(st *state.State, runner *state.TaskRunner) *ServiceManager {
 
 	// TODO: undo handler
 	runner.AddHandler("quota-control", m.doQuotaControl, nil)
-
+	AddAffectedQuotasByKind("quota-control", quotaControlAffectedQuotas)
 	snapstate.AddAffectedSnapsByKind("quota-control", quotaControlAffectedSnaps)
 
+	// A separate, purpose specific task to handle support for adding
+	// a snap to a quota group upon install. We can't directly refer to
+	// the servicestate package, so this task takes care of calling quotaUpdate
```

Comment:
It's unclear what this comment means, because we already are inside the `servicestate` package, so there should be no issue in using any of its members.